bikeshed.js

```diff
@@ -97,7 +105,7 @@ function updateBikes() {
 
     bikeList.innerHTML = '';
 
-    database.items.forEach(function (bike) {
+    database.getItems.forEach(function (bike) {
```

Comment:
Har du testet at det virker? Jeg ville tro at der skulle stå `database.getItems()`

Reply:
Før var `items` jo et felt i `database`, og så er et fint at skrive `database.items`, men nu kalder du en funktion og så forventer jeg at du skal skrive `database.getItems()`

Reply:
Det virker.
_"In general, getter methods do not need to be called with a set of parentheses. Syntactically, it looks like we’re accessing a property"_

Reply:
Nice. Så er alt godt :)